panda/src/bullet/bulletVehicle.h

```diff
@@ -85,7 +85,7 @@ class EXPCL_PANDABULLET BulletVehicle : public TypedReferenceCount {
   void apply_engine_force(PN_stdfloat force, int idx);
 
   // Wheels
-  BulletWheel create_wheel();
+  BulletWheel create_wheel(PN_stdfloat suspension_rest_length);
```

Comment:
I think this should have a default value of `0.4` so that it doesn't break existing code.

Reply:
I've added a default in [this commit](https://github.com/panda3d/panda3d/pull/569/commits/7412688aa916a9539418c941cf3d50b0ef94b036)